.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report
+about: Report problems with the OpenDI site, its content, or its deployment processes
+title: 'Bug: [Brief title here]'
+labels: bug, community submission
+assignees: ''
+
+---
+
+## Reminders
+(To save space, you may delete this section of the template before submitting your issue)
+
+Please submit your feedback to the most relevant repository:
+- [opendi-org/landing-site](https://github.com/opendi-org/landing-site) for general OpenDI feedback, or feedback related to the OpenDI homepage and related content
+- [opendi-org/roles-user-stories](https://github.com/opendi-org/roles-user-stories) for feedback related to the OpenDI roles and user stories
+- [opendi-org/json-schema](https://github.com/opendi-org/json-schema) for feedback related to the OpenDI JSON Schema
+
+**Always use the Issue search feature to see if your report, request, or feedback already exists!**  
+If your issue already exists, add your voice to the comments section! Provide your unique perspective, additional info, etc.
+
+Please fill in the remaining portions of the template to the best of your ability. This will help us get the most value out of your contributions!
+
+## Describe the bug
+> A clear and concise description of what the bug is.
+
+## Bug category
+> Which category best fits this bug? Please indicate one of the following:
+> - **Content**: Bug concerns the standards content itself (inconsistency, typo, unclear, etc.)
+> - **Site**: Bug concerns the public website (broken link, error 404/missing content, etc.)
+> - **Deployment**: Bug concerns the deployment process (GitHub Actions, mkdocs generation process or configuration, etc.)
```

Comment:
This won't hold up the approval, but we may want to consider editing this section to have repo-specific deployment tool chains (e.g., MKDocs vs. others).